package.json

```diff
@@ -30,12 +31,12 @@
     "koa-compress": "^1.0.9",
     "koa-handlebars": "^0.5.7",
     "koa-helmet": "^1.0.0",
-    "koa-request": "^1.0.0",
+    "koa-request": "1.0.0",
     "koa-response-time": "^1.0.2",
     "koa-router": "^5.4.0",
     "koa-static": "^2.0.0",
-    "moment-timezone":"0.5.3",
-    "koa-request":"1.0.0"
+    "moment-timezone": "0.5.3",
+    "request": "^2.72.0"
```

Comment:
@ianrose this should be a dev dep as its only used with grunt. I'm planning to switch to Gulp also.